cdap-elastic/src/test/java/io/cdap/cdap/metadata/elastic/ElasticsearchMetadataStorageTest.java

```diff
@@ -494,6 +494,12 @@ public void testNumericQueries() throws IOException {
     Assert.assertEquals(records.get(1), response6.getResults().get(0));
     Assert.assertEquals(records.get(2), response6.getResults().get(1));
 
+    // test that decimal format is supported
+    SearchRequest request7 = SearchRequest.of("p:=0.0").build();
```

Comment:
This syntax reminds me of an assignment like in `x := 1` in some programming languages.  Perhaps it makes more sense to use `==` instead of `=`?

Reply:
A reasonable suggestion. `==` is more reminiscent of the boolean syntax, which I think is more closely related to what a search query is looking for.
This comment has been addressed in the latest commit. The requisite changes have also been made on the query parsing branch.